dashboard/config/locales/slides.en.yml

```diff
@@ -2451,3 +2451,76 @@ en:
+      2:
+        image: 'notes/writing_first_program_2.png'
+        text: "Each puzzle has four parts."
+      3:
+        image: 'notes/writing_first_program_3.png'
+        text: "The play area is on the left, where your program runs."
+      4:
+        image: 'notes/writing_first_program_4.png'
+        text: "The center has the toolbox area, which has all the blocks you need."
+      5:
+        image: 'notes/writing_first_program_5.png'
+        text: "On the right is the workspace, where you will drag blocks to build your program."
+      6:
+        image: 'notes/writing_first_program_6.png'
+        text: "Above the workspace are the instructions for each puzzle."
+      7:
+        image: 'notes/writing_first_program_7.png'
+        text: "If you drag out too many blocks from the toolbox, just drag the extra block back into the toolbox.  Hit run to try your program, and reset to try again."
+    repeat_block:
+      1:
+        image: 'notes/repeat_block.png'
+        text: "A repeat block lets you loop the code inside of it as many times as you tell it to."
+    what_is_an_event:
+      1:
+        image: 'notes/what_is_an_event.png'
+        text: "An event tells your program to listen for something to happen. When that thing happens, the program performs an action. A computer might listen for a mouse click, press on an arrow button, or a tap on the screen."
+    lesson_planning:
+      1:
+        image: 'notes/lesson_planning_1.png'
+        text: "CS Fundamentals lessons begin with overviews to let you know what the lesson will cover."
+      2:
+        image: 'notes/lesson_planning_2.png'
+        text: "Lessons have warm ups, activities, and wrap ups. If you want to create your own lesson plans, you might want to keep these tips in mind."
+      3:
+        image: 'notes/lesson_planning_3.png'
+        text: "1. Try to use a spiraling curriculum design. Introduce concepts once and then revisit them."
+      4:
+        image: 'notes/lesson_planning_4.png'
+        text: "2. Create a blended learning experience by combining online lessons with unplugged lessons."
+      5:
+        image: 'notes/lesson_planning_5.png'
+        text: "3. Break your curriculum into smaller chunks."
+      6:
+        image: 'notes/lesson_planning_6.png'
+        text: "4. Make sure that you align to local standards and college and career readiness skills."
+    opd_closing:
+      1:
+        image: 'notes/opd_closing_1.png'
+        text: "Congratulations! You completed the Teaching Computer Science Fundamentals online course. You’re now part of the growing community of teachers preparing their students for the future."
+      2:
+        image: 'notes/opd_closing_2.png'
+        text: "Check out these resources if you need more support: 1. forum.code.org 2. @teachcode on Twitter 3. studio.code.org "
```

Comment:
nit: is there any way to get newlines into this text?

Reply:
@breville I tried different options to add new-lines and did not find an option that worked.

https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines/21699210#21699210

This might be due to how the strings in slides.en.yml are parsed.

Reply:
Ah yes, it seems not worth pursuing this further at the moment.  I was able to get some newlines from the YML into the HTML using a technique from the article you linked, but those newlines are not rendered as such in HTML, unless we do something like add `white-space: pre-wrap;` to the `<p>`s.